uqcsbot/whatweekisit.py

```diff
@@ -2,6 +2,7 @@
 from discord import app_commands
 import requests
 from typing import NamedTuple, List, Tuple, Optional
+from zoneinfo import ZoneInfo
```

Comment:
in theory you should be able to get away with using the existing datetime imports: replace your ZoneInfo call with `timezone(timedelta(hours=10), "AEST")`

Reply:
Gonna disagree as I do prefer the `Australia/Brisbane` definition as it's a tiny bit clearer (alas an extra import, but we already do this for pytz.